proposal.md

```diff
@@ -14,28 +15,38 @@ Summary
 
 
 > "Four shalt thou not count, neither shalt thou count two, excepting that thou
-then proceedeth to three."
+then proceed to three."
```

Comment:
I checked this important point [with Youtube](https://www.youtube.com/watch?v=xOrgLj9lOwk). It is 'proceed'.